src/microbe_stage/CompoundBag.cs

```diff
@@ -244,4 +244,13 @@ IEnumerator IEnumerable.GetEnumerator()
     {
         return GetEnumerator();
     }
+
+    public CompoundBag Clone()
```

Comment:
Shouldn't this clone specific capacities as well?

Reply:
Due to a change in how stuff is done, I just undid this change.